pkg/controller/jobs/pod/pod_webhook.go

```diff
@@ -190,20 +179,23 @@ func (w *PodWebhook) Default(ctx context.Context, obj runtime.Object) error {
 		}
 
 		suspend = jobframework.QueueNameForObject(pod.Object()) != "" || w.manageJobsWithoutQueueName
+		if suspend {
```

Comment:
Why is this moved?

Reply:
https://github.com/kubernetes-sigs/kueue/pull/4808/files/42a03da0c49ed54d38f1f198f25a05ce19dd7646#r2039303467

Reply:
https://github.com/kubernetes-sigs/kueue/pull/4808/files/42a03da0c49ed54d38f1f198f25a05ce19dd7646#r2039303467

Reply:
https://github.com/kubernetes-sigs/kueue/pull/4808#discussion_r2039303467

Reply:
Ok, got it. Thanks